test/integration/generated/url/Source/AutoRestUrlTestClient.swift

```diff
@@ -68,9 +68,9 @@ public final class AutoRestUrlTestClient: PipelineClient {
     // /// should contain value null
     public var globalStringQuery: String?
 
+    public lazy var queries: Queries = Queries(client: self)
     public lazy var paths: Paths = Paths(client: self)
     public lazy var pathitems: PathItems = PathItems(client: self)
```

Comment:
I'm curious why the properties, upon being regenerated, change the ordering. There may be somewhere we are iterating through a collection that doesn't guarantee order. Not important for this PR, but it would be great to identify this and correct.

Reply:
Add code to sort the shortcuts in Service Client Model.